R/removeTemporal.cf.R

```diff
@@ -180,7 +180,7 @@ old_removeTemporal.cf <- function(cf,
 #' @return
 #' The shifted correlator as an object of type `cf`, see \link{cf}
 #'
-#' @eport
+#' @export
```

Comment:
This is of course a needed fix. But keep in mind that it does not have any effect at this point because of our hand-written `NAMESPACE` file.